docs/sources/mimir/references/http-api/index.md

```diff
@@ -523,20 +523,23 @@ Requires [authentication](#authentication).
 GET,POST <prometheus-http-prefix>/api/v1/cardinality/label_names
 ```
 
-Returns realtime label names cardinality across all ingesters, for the authenticated tenant, in `JSON` format.
+Returns label names cardinality across all ingesters, for the authenticated tenant, in `JSON` format.
 It counts distinct label values per label name.
 
 As far as this endpoint generates cardinality report using only values from currently opened TSDBs in ingesters, two subsequent calls may return completely different results, if ingester did a block
 cutting between the calls.
 
 The items in the field `cardinality` are sorted by `label_values_count` in DESC order and by `label_name` in ASC order.
-
 The count of items is limited by `limit` request param.
 
 This endpoint is disabled by default and can be enabled via the `-querier.cardinality-analysis-enabled` CLI flag (or its respective YAML config option).
 
 Requires [authentication](#authentication).
 
+#### Caching
+
+The response of this API endpoint could be fetched from the query-frontend results cache if `-query-frontend.cache-results` is enabled and `-query-frontend.results-cache-ttl-for-cardinality-query` set to a value greater than 0.
```

Comment:
```suggestion
The response of this API endpoint can be fetched from the query-frontend results cache if `-query-frontend.cache-results` is enabled and `-query-frontend.results-cache-ttl-for-cardinality-query` set to a value greater than `0`.
```
Who is performing these actions? Please reword the passive voice if possible.

Reply:
How does this sound?

```suggestion
The query-frontend can return a stale response fetched from the query results cache if `-query-frontend.cache-results` is enabled and `-query-frontend.results-cache-ttl-for-cardinality-query` set to a value greater than `0`.
```